metis/spec/client_spec.rb

```diff
@@ -809,5 +834,22 @@ def token_with_exp(exp)
 
       FileUtils.rm_r('athena-copy')
     end
+
+    it 'only looks within the specified project' do
+      bucket = create( :bucket, project_name: 'athena', name: 'armor', access: 'editor', owner: 'metis')
+
+      helmet_file = create_file('athena', 'helmet.jpg', HELMET, bucket: bucket)
+      stubs.create_file('athena', 'armor', 'helmet.jpg', HELMET)
+      folly_file = create_file('ate', 'folly.txt', WISDOM.reverse, bucket: bucket)
```

Comment:
Hm, does this do what you intend (create another file in a project called `ate`)? I'm not sure `create_file` will create the project and generate a `Metis::File` record for it? Plus, wouldn't you need to have the `bucket` in that case, belong to the specific project?

Perhaps a good counter-test would be to have the exact same setup code and queried without the `--project-name athena` flag, and see if the `folly.txt` file is returned...

I believe this idea would apply to all the new tests.

Reply:
Hmm, the bucket is definitely wrong here. I think it will work anyway because the actual check only relies on the `project_name` string on file, otherwise there is no project entry. I added some more fixtures to the default cross-project search for good measure.